app/controllers/admin/collections_controller.rb

```
@@ -43,7 +43,7 @@ def destroy
       if unlocked_collection.destroy
         flash[:notice] = t('.deleted')
       else
-        flash[:alert] = t('.not_empty_error')
+        flash[:alert] = unlocked_collection.errors.full_messages.first
```

Let's bubble up the actual error from the model and show this? Instead of a generic error message instead?